datalab/datalab_session/analysis/source_catalog.py

```diff
@@ -0,0 +1,37 @@
+from datalab.datalab_session.util import get_hdu, scale_points
+import numpy as np
+
+
```

Comment:
Lots of extra newlines in this file - we try and follow pep8 in this case. https://peps.python.org/pep-0008/#blank-lines